app/views/devise/sessions/new.html.erb

```diff
@@ -5,7 +5,7 @@
 
 <h2>Log in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name), class: "session") do |f| %>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
   <div class="field">
     <%= f.label :email, class: "form-label" %><br />
     <%= f.email_field :email, autofocus: true, autocomplete: "email",
```

Comment:
- [ ] Nice job making the signup/login page work 👌 However, this activity requires the modification of the views in _devise_ for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly, as seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your login view:**        |  **Wireframe login view:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/juaryimami/blog-app/assets/12745474/bf410121-caa7-496f-90e9-c871186de354"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png">